### DISCUSSION TITLE
Hydration completed but contains mismatches Nuxt3

### DISCUSSION THREAD

**Discussion by Nothehi** [nuxt/nuxt#18401] [Questions]
I'm trying to build an SSG website and deploy it to GitHub Pages, but I'm getting this error:
![image](https://user-images.githubusercontent.com/43233476/213667451-19dda754-f912-41d2-b28e-2e2c0a1c5faf.png)
even in my local when I run `npm run generate` and open the generated file I have these errors:
![image](https://user-images.githubusercontent.com/43233476/213668878-4c540fd7-eac1-4187-abdb-3a9f03096eb7.png)

I have just one page with simple vue and html code. Tried many ways to fix this problem but still, the problem is there! this is my nuxt info:
- Operating System: `Windows_NT`
- Node Version:     `v18.13.0`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@9.3.0`
- Builder:          `vite`
- User Config:      `modules`, `ssr`, `experimental`, `app`
- Runtime Modules:  `@nuxtjs/tailwindcss@6.1.3`
- Build Modules:    `-`

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

**Reply by Nothehi** [reply]
Because I am using SSG, I am not sure if the exact same problem occurs in StackBlits or CodeSandBox, but if it helps [this ](https://github.com/Nothehi/cv) is my Git repository.

**Reply by TheAlexLichter** [reply]
You can provide a StackBlitz/CodeSandbox that you can run via ´npm run generate` and then serving the files. That's more than fine too ☺️

**Reply by TheAlexLichter** [reply]
But in this case (as the project is still quite small) I'll have a look

**Reply by TheAlexLichter** [reply]
I can't reproduce. Did you maybe open the files via "file explorer" instead of serving them via server?

**Reply by Nothehi** [reply]
The error in the second image is related to when I open the generated file after running the generate command via file explorer, but the first image is related to the GitHub server. I don't know why I'm getting a hydration error on GitHub pages!

**Reply by Nothehi** [reply]
I tried to create a [new project](https://colloc.ir/) and start again but now I have two repo with the same hydration error! why does this happen?

**Reply by TheAlexLichter** [reply]
This is most likely because of your deployment process. Building locally for production, things work fine.
